### DISCUSSION TITLE
All sent emails end up in spam

### DISCUSSION THREAD

**Discussion by RoderikTheRed** [modoboa/modoboa#2593] [Help]
After setting up Modoboa server, I noticed that all the mails I send from the server/domain end up in the recipient's spam filter.

I think all of my DNS Entrys are OK, as its all green in the Dashboard.
I am using Cloudflare to manage the DNS Entrys could that be a problem?

![Screenshot 2022-09-05 153446](https://user-images.githubusercontent.com/66172264/188462394-bde31685-06e0-4900-b963-e652f848d5f6.png)

**Comment by RoderikTheRed** [comment]
Update: I tested around and added a DKIM record, as I though this could be the issue.

Still no improvement, I am honestly out of ideas.
Also switched to the new Admin overlay

If you need any additional Information, I'll be more than happy to provide it to you.

![Screenshot 2022-09-05 215223](https://user-images.githubusercontent.com/66172264/188507152-cbcc1f02-c602-4638-b71e-b23ac352cb9a.png)

**Comment by tonioo** [comment]
@RoderikTheRed Have you access to sent emails (on the recipient side). Sometimes you can find useful headers in messages indicating why they have been classified as spam.
